src/rust/engine/README.md

```diff
@@ -4,12 +4,7 @@ A native-code implementation of the pants v2 engine. See:
 
 To build for development, simply run pants from the root of the repo:
 
-    ./pants list 3rdparty::
-
-If you plan to iterate on changes to the rust code, it is recommended to set `MODE=debug`, which
```

Comment:
Would be good to still have this explanation, but inverted?

Reply:
Perhaps, but I plan to migrate most of these docs to https://pants.readme.io/docs/contributions-rust. (Why? More discoverable to new people + centralizes information). 

We mention already in https://pants.readme.io/docs/contributor-setup#step-3-bootstrap-the-rust-engine that you should set `MODE=release` if you're doing lots of Python work.